index.d.ts

```diff
@@ -294,6 +294,7 @@ interface EventMp {
 }
 
 interface ConfigMp {
+	[prop: string]: any,
```

Comment:
What should this be for? Wouldn't this mean that ConfigMp can have any property name - even though there is a fixed set of config options?

Or does this mean that every config property in the JSON object in the file `conf.json` is available? I do not know if the server deserializes non-fixed values into that object.

Reply:
You can add in conf.json what ever you want and read it via mp.config so this line makes sense
I use it in a lot of my scripts - so thought I should mention that

Reply:
Good to know.